app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_secure_password
+
+  validates :name, presence: true
+  validates :email, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP }, uniqueness: { case_sensitive: false }
+  validates :password, length: { minimum: 6 }
```

Comment:
Will it validate presence of password?

Reply:
No bhai, To validate presence of password. presence: true should be add.